src/selectors.js

```diff
@@ -74,29 +94,84 @@ const _buildRecords = (ents, key, opts) => {
 
 const defaultArr = [];
 
-const buildRecords = (ents = {}, key, opts = {}) => {
-  // console.log(`All. key: ${key}`, ents, opts)
-  if (!_.isObject(opts)) opts = {};
-  const { model, defaultJoins } = getModel(key);
-  const joinKeys = utils.composeArray(opts.joins, defaultJoins);
-  let final = enrichRecords(ents, key) || defaultArr;
-
-  joinKeys.forEach((k) => {
-    const mergeItems = enrichRecords(ents, k);
-    final = joins({
-      items: final,
-      mergeItems,
+const buildRecords = (entities = {}, key, opts = {}) => {
```

Comment:
lijkt me goed om deze fn op te delen, bijv. om een paar loops er uit te halen. Nu is het wel erg complex